src/main/java/racingcar/Game.java

```diff
@@ -0,0 +1,55 @@
+package racingcar;
+
+import java.util.Random;
+
+public class Game {
+    private static int roundCount;
+    private static Car[] cars;
+    private static Random random = new Random();
+
+    public static void setRoundCount(int count){
+        roundCount = count;
+    }
```

Comment:
```suggestion
    public static void setRoundCount(int roundCount){
        this.roundCount = roundCount;
    }
```
이렇게 일치시켜주면 좋을 것 같습니다!